docs/compression/README.md

```diff
@@ -0,0 +1,76 @@
+# Compression Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+Minio server has compression disabled by default. By enabling it, Minio ensures efficient disk space usage by compressing the objects before being written to disk(s). 
+
+Minio adopts streaming compression format of `golang/snappy`. For better performance, Minio compresses only `compressible objects`. These are objects which will produce a better compression experience. For instance, already compressed objects like `.zip,.bz2` etc, are not fit for compression since they do not have compressible patterns in them.
+
+
+Compression is disabled for
```

Comment:
looks like the sentence is half complete..

Reply:
modified the docs fully